tests/test_benchmarks.py

```
@@ -0,0 +1,537 @@
+            return result.seconds_remaining
+
+        result = benchmark(estimate)
+        assert result >= 0
+
+    def test_benchmark_get_rule_estimate(
+        self, benchmark: BenchmarkFixture, populated_estimator: TimeEstimator
+    ) -> None:
+        """Benchmark per-rule estimation."""
+
+        def get_estimate() -> Any:
+            duration, variance = populated_estimator.get_estimate_for_job(
```

Ruff (0.14.10)

Unpacked variable `variance` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)